VowelConsonant.java

```diff
@@ -0,0 +1,30 @@
+// to count total number of vowels and consonants
+
+import java.util.Scanner;
+
+public class VowelConsonant {
+    public static void main(String[] args) {
+        int vowelCount = 0;
+        int consonantCount = 0;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the string : ");
+        String input = sc.nextLine();
+        input = input.toLowerCase();
+        char[] stringArray = input.toCharArray();
+        char[] vowels = {'a', 'e', 'i', 'o', 'u'};
+        for (int j = 0; j < stringArray.length; j++) {
```

Comment:
logical error please test it before creating PR.

Reply:
done